TCAT/Model/Route.swift

```diff
@@ -144,14 +147,17 @@ class Route: NSObject, JSONDecodable {
      * Modify the first routeSummaryObject if the name is "Start"
      *  If the starting place is a place result or current location
      *      AND the route summary array count is more than 2 (has a route that is more than simply walking)
-     *      remove the first routeSummaryObject
+     *      remove the first routeSummaryObject and update the time
      *  Else (the first routeSummaryObject is a bus stop), so simply update the name to the name the user searched for
      */
     func updateStartingDestination(_ place: Place) {
         if let firstRouteSummaryObject = routeSummary.first {
             if firstRouteSummaryObject.name == "Start" {
                 if (place is PlaceResult || (place is BusStop && place.name == "Current Location")) && routeSummary.count > 2 {
                     routeSummary.remove(at: 0)
+                    departureTime = (routeSummary.first?.time)!
```

Comment:
Is this force unwrap ok? Will it always always work?

Reply:
yes b/c i check `routeSummary.count > 2` in if statement so `routeSummary.first?` will never be nil and `time` is not an optional property in `RouteSummaryObject`

Reply:
but agree this not good style >.<